sound/soc/sof/topology.c

```diff
@@ -1920,7 +1920,8 @@ static void sof_dai_set_format(struct snd_soc_tplg_hw_config *hw_config,
 /* set config for all DAI's with name matching the link name */
 static int sof_set_dai_config(struct snd_sof_dev *sdev, u32 size,
 			      struct snd_soc_dai_link *link,
-			      struct sof_ipc_dai_config *config)
+			      struct sof_ipc_dai_config *playback,
+			      struct sof_ipc_dai_config *capture)
```

Comment:
@RanderWang I think this change looks rather awkward. You really only use both playback and capture configs for SSP dai links and in that case both of them are the same.